### ISSUE TITLE
chinese punctuation Inaccurate matching

### ISSUE THREAD

**Issue by wangzaogen** [JaidedAI/EasyOCR#162]
hi Jaided

I have chinese pic  include :  ‘孤山寺北贾亭西，水面初平云脚低。’

easyocr  Output : ['钱塘湖春行', '原文:', '孤山寺北贾亭西,水面初平云脚低6', '几处早莺争暖树,谁家新燕啄春泥6', '乱花渐欲迷入眼,浅草才能没马蹄6', '最爱湖东行不足,绿杨阴里自沙堤6']

the word is very exact matches,
but punctuation has a problem `。`  > ` 6`，want to match `。` > ` 。`
How to  advance it?
thanks

**Comment by rkcosmos** [comment]
That's because I didn't include  `。` in the character list. I'll add it to the next model. From [Wikipedia](https://en.wikipedia.org/wiki/Chinese_punctuation), there are several other Chinese punctuation 「...」,  ﹁...﹂, 《…》. Are they all used in modern text?

**Comment by wangzaogen** [comment]
yes. it's very well . if the character list include all  Chinese punctuation, then  easyocr matching Chinese is very accurate.
I can give you some punctuation
```
。，、＇：∶；?‘’“”〝〞ˆˇ﹕︰﹔﹖﹑·¨….¸;！´？！～—ˉ｜‖＂〃｀@﹫¡¿ 々﹟#﹩$﹠&﹪%*﹡﹢﹦﹤‐￣¯―﹨ˆ˜﹍﹎+=<­­＿_-\ˇ~﹉﹊（）〈〉‹›﹛﹜『』〖〗［］《》〔〕{}「」【】
```
you can add it to the next model.

**Comment by wangzaogen** [comment]
when you new versions of easyocr is released, i can test the chinese punctuation problem

**Comment by rkcosmos** [comment]
sure, I'll tell you when it's done. It's going to take a while because I'm focusing on expanding support to other languages.

**Comment by wangzaogen** [comment]
By the way，do you have a plan?  write a wiki for easyocr.

**Comment by rkcosmos** [comment]
never thought about that before, it's a good idea.

**Comment by rkcosmos** [comment]
new model released in version 1.3.0.1.
